src/preferences/dialog/dlgprefsound.cpp

```diff
@@ -405,7 +403,7 @@ void DlgPrefSound::insertItem(DlgPrefSoundItem *pItem, QVBoxLayout *pLayout) {
     for (pos = 0; pos < pLayout->count() - 1; ++pos) {
         DlgPrefSoundItem *pOther(qobject_cast<DlgPrefSoundItem*>(
             pLayout->itemAt(pos)->widget()));
-        if (!pOther) {
+        if (!pOther) { // don't replace existing. DEBUG_ASSERT ?
```

Comment:
Please decide, if this should be an assert. If not remove the comment.

Reply:
:+1:

Reply:
didn't look closer earlier, this could theoratically happen if we added a spacer or separator, which is not the case currently